### PR TITLE
docs(hide): add hide comments

### PR DESCRIPTION
**Pull request by mtomcal**
- We have added Hide and Metadata magic comments in One Docs
- Need to add the comments here to remove markdown not necessary to a web experience.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +19,7 @@ Want to get paid for your contributions to `one-app-ducks`?
 
 * [Features](#-features)
 * [Usage](#-usage)
-* [API](#%EF%B8%8F-api)
+* [API](#%EF%B8%8F-one-app-ducks-api)
```
Comment (JamesSingleton) [inline README.md:22]: Why did this change?

Comment (mtomcal) [reply README.md:22]: Changed the API title to `One App Ducks API` for web experience purposes

Comment (JamesSingleton) [reply README.md:22]: Is this something that will be happening across all repos?

Comment (mtomcal) [reply README.md:22]: We will need to add Metadata comments to all Markdowns in order to have correct tags, title and authorship.

Some libraries like Iguazu that only have a single README will require Hide tags added to remove unnecessary Markdown sections like "Contributing" for the web experience.

**Pull request merged**